samcli/local/apigw/service.py

```diff
@@ -366,6 +371,37 @@ def _construct_event(flask_request, port, binary_types):
         LOG.debug("Constructed String representation of Event to invoke Lambda. Event: %s", event_str)
         return event_str
 
+    @staticmethod
+    def _query_string_params(flask_request):
+        """
+        Constructs an APIGW equivalent query string dictionary
+
+        Parameters
+        ----------
+        flask_request request
+            Request from Flask
+
+        Returns dict (str: str)
+        -------
+            Empty dict if no query params where in the request otherwise returns a dictionary of key to value
+
+        """
+        query_string_dict = {}
```

Comment:
nit: Most variables in this method are named `query_string_<type>` or some flavor of it. This is not descriptive. You don't need the prefix because you are inside a method that processes query string parameters